src/libraries/System.Private.CoreLib/src/System/Numerics/IUnsignedNumber.cs

```diff
@@ -6,7 +6,8 @@ namespace System.Numerics
     /// <summary>Defines a number type which can only represent positive values, that is it cannot represent negative values.</summary>
     /// <typeparam name="TSelf">The type that implements the interface.</typeparam>
     public interface IUnsignedNumber<TSelf>
-        where TSelf : INumberBase<TSelf>, IUnsignedNumber<TSelf>
+        : INumberBase<TSelf>
+        where TSelf : IUnsignedNumber<TSelf>
```

Comment:
Can you explain this change?

Reply:
The constraint, given the expected usage that `TSelf` is the thing implementing `IUnsignedNumber`, was being unnecessarily "more verbose" and required downstream constrainers to specify "more" than necessary in many cases.

This ensures that devs can just constrain to `where T : IUnsignedNumber<T>` and not have to worry about or deal with `n` other constrains as well.